app/jobs/sms_account_reset_cancel_job.rb

```diff
@@ -0,0 +1,14 @@
+class SmsAccountResetCancelJob < ApplicationJob
+  queue_as :sms
+  include Rails.application.routes.url_helpers
```

Comment:
What are the `url_helpers` used for here?

Reply:
Leftover from a copy and paste of the other job that needed it for account_reset_cancel_url.  I'll remove it.